Makefile

```diff
@@ -1,5 +1,5 @@
 #TODO add default features here
-export FEATURES?=sctp performance xt_u32
+export FEATURES?=sctp performance xt_u32 cmns
```

Comment:
I understand why you added that here
but probably need to remove from default before merging